### ISSUE TITLE
v8.7 branch broken with tip of Coq v8.7 branch

### ISSUE THREAD

**Issue by Zimmi48** [ejgallego/coq-serapi#39]
SerAPI builds with coq/coq@4b7e1cc but not with coq/coq@bc5cdc6.

**Comment by ejgallego** [comment]
What kind of error you get, is it due to ocaml-ppx/ppx_import#17 ?

If that is the case, we cannot do anything until the problem is solved upstream. The PPX ecosystem is still extremely unstable :( [should that serve as advice for Coq]

I really do wonder why they had to introduce a fatal error in this case instead of just a warning, it seems not sensible to me.

**Comment by Zimmi48** [comment]
```
File "serlib/ser_vernacexpr.ml", line 248, characters 39-61:
Error: Unbound type constructor cumulative_inductive_parsing_flag
```

So I guess this is unrelated (only due to a change of API in Coq).

**Comment by ejgallego** [comment]
I have a fix for that, will push, [sadly untested until I resolve the ppx issues]

**Comment by Zimmi48** [comment]
Fixed by b31e638de32b2624d419cb6fb2aaa5d2c4d9d5e2.

**Comment by ejgallego** [comment]
I traced the ppx problems back to `ppx_deriving` 4.2. Downgrading to 4.1 solves the problem.
